mappings/net/minecraft/world/chunk/WorldChunk.mapping

```diff
@@ -58,11 +58,20 @@ CLASS net/minecraft/class_2818 net/minecraft/world/chunk/WorldChunk
 		ARG 1 levelTypeProvider
 	METHOD method_12208 createBlockEntity (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2586;
 		ARG 1 pos
-	METHOD method_12210 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+	METHOD method_12210 collectEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/List;Ljava/util/function/Predicate;)V
+		COMMENT Collects a list of entities of some runtime type and appends them to the given list. The type of
+		COMMENT the collected entities will be a subclass of the given class.
+		COMMENT
+		COMMENT @throws IllegalArgumentException If {@code entityClass} is an interface or is not a subclass
```

Comment:
Hmm, so `Mob` or `Hoglin` interface etc cannot be used? Good note

Reply:
That is correct, the method eventually calls `TypeFilterableList#getAllOfType`, which checks for compatibility using `Class#isAssignableFrom`.